tests/lib/external/snapd-testing-tools/tools/os.query

```diff
@@ -60,6 +62,16 @@ is_core22() {
     fi
 }
 
+is_core24() {
```

Comment:
Hi, this needs to be included in the snapd-testing-tools project

Reply:
https://github.com/snapcore/snapd-testing-tools/pull/38

Reply:
I think you just need to merge master because it is on master

Reply:
Done.